docker/service/src/main/java/uk/nhs/adaptors/scr/services/GetScrService.java

```diff
@@ -129,12 +131,12 @@ private Bundle buildBundle() {
     @LogExecutionTime
     public Bundle getScr(String nhsNumber, String compositionId, String nhsdAsid, String clientIp) {
         Document scrIdXml = getScrIdRawXml(nhsNumber, nhsdAsid, clientIp);
-        LOGGER.debug("Received SCR ID XML:\n{}", scrIdXml);
+        logXml("Received SCR ID XML: {}", scrIdXml);
```

Comment:
Might be better to log that on a lower level where it is still a string. This way you'll always see the xml even if it's malformed and could not be parsed. It will also remove the risk where parsing/serializing could add some unwanted data

Reply:
I've chosen debug for 2 reasons:
1) it might contain sensitive data that should not be logged
2) performance

Reply:
Ok, we discussed this offline